biscuit-auth/src/format/schema.proto

```diff
@@ -214,6 +215,8 @@ message AuthorizerPolicies {
 message ThirdPartyBlockRequest {
   required PublicKey previousKey = 1;
```

Comment:
we should rename it as `legacy_public_key` (not a breaking change since the field id does not change)